sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_HealthcareAsync_Cancellation.cs

```diff
@@ -32,10 +33,16 @@ public async Task HealthcareAsyncCancellation()
                                 minimal ST depressions in the anterior lateral leads , thought due to fatigue and wrist pain , his anginal equivalent. Due to the patient's \
                                 increased symptoms and family history and history left main disease with total occasional of his RCA was referred for revascularization with open heart surgery.";
 
-            HealthcareOperation healthOperation = await client.StartHealthcareAsync(document);
+            var batchDocument = new List<string>();
 
-            await client.StartCancelHealthJobAsync(healthOperation);
+            for (var i=0; i < 10; i++)
```

Comment:
Could you also change the tests for pagination? https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs#L266